spec/models/hyrax/collection_type_spec.rb

```diff
@@ -21,18 +21,18 @@
   end
 
   describe '#gid' do
-    it 'returns the gid when id is not nil' do
+    it 'returns the gid when id exists' do
       collection_type.id = 5
       expect(collection_type.gid.to_s).to eq 'gid://internal/hyrax-collectiontype/5'
     end
 
-    it 'returns the gid when id is nil' do
+    it 'returns nil when id is nil' do
       collection_type.id = nil
       expect(collection_type.gid).to be_nil
     end
   end
 
-  describe ".find_or_create_default_collection_type" do
+  describe "#find_or_create_default_collection_type" do
```

Comment:
This should be `. find_or_create_default_collection_type` (as it is a class method). Instance methods are documented as `#find_or_create_default_collection_type`.